### ISSUE TITLE
Scores and Rewards

### ISSUE THREAD

**Issue by mschilling** [mschilling/codeheroes-service#7]
We should have a transparent overview of scores and rewards.

Currently, the following scores apply:
* commit: +1
* push: +1
* pull request: +3
* issue opened: +5
* issue closed: +2

Suggestions are welcome. For example, 5 points for opening a github issue is probably a bit too much.. right?

**Comment by wwolters** [comment]
I do not agree with awarding points for pushes, as that would allow people to easily double their points by just pushing after every commit.
Maybe pull requests should only award points when they're accepted.
Points for opening an issue should probably only be awarded when the issue is accepted by the repo owner somehow, maybe when a certain label is applied?

**Comment by mschilling** [comment]
> I do not agree with awarding points for pushes, as that would allow people to easily double their points by just pushing after every commit.

Fair point.. :-)

> Points for opening an issue should probably only be awarded when the issue is accepted by the repo owner somehow, maybe when a certain label is applied?

To motivate developers to take the time to add an issue, it should at least be rewarded by one or more points. But, depending on the quality (e.g. approved by repo owner or votes) it could be possible to get 'more' points. 

We should find a way to start with a really simple counter/scores model and improve it on the go (Y)

**Comment by mschilling** [comment]
` const SCORE_CONSTANTS = {
  push: 0,
  commit: 1,
  pullRequest: 3,
  githubIssueOpened: 2,
  githubIssueClosed: 2
};`

updated scores..

**Comment by mschilling** [comment]
Improved scores for pull request (actions:
* pull request - opened: +2
* pull request - closed: +3
* pull request - other: +0
